src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameFixes.java

```diff
@@ -162,12 +174,13 @@ public void onPistonPushRail(BlockPistonExtendEvent event)
 			for (Block block : event.getBlocks())
 			{
 				Material type = block.getType();
-				if(type == Material.RAILS ||
-					type == Material.ACTIVATOR_RAIL ||
-					type == Material.DETECTOR_RAIL ||
-					type == Material.POWERED_RAIL)
+				for (Material mat : stopDupes_)
```

Comment:
same; `if (stopDupes_.contains(mat)) {`

assuming you change stopDupes from a flat array to a structured Set, ofc.